dsl/src/test/scala/com/crobox/clickhouse/dsl/language/LogicalFunctionTokenizerTest.scala

```diff
@@ -125,9 +125,17 @@ class LogicalFunctionTokenizerTest extends ClickhouseClientSpec with TestSchema
     )
   }
 
+  it should "tokenize numbers OR with NONE" in {
```

Comment:
Add queries that hit the `isConstFalse` and `isConstTrue` codepaths?

Reply:
Done